### ISSUE TITLE
Make Custom Banner for the Competition balcony

### ISSUE THREAD

**Issue by lukasmwerner** [LaSalleRobots/PowerPlay#34]
Talk to Coleman about using Large Format Printer to print something and then probably back the paper with some tape? idk this should be pretty cost effective?

**Comment by pdxaeroman** [comment]
Will they let us put it up there? And why?

**Comment by lukasmwerner** [comment]
Yes, in fact they asked us to put one up next time (along with all hands on tech)

**Comment by pdxaeroman** [comment]
I guess we're part of the cool kids club then?

**Comment by pdxaeroman** [comment]
Do we have a large format laminator anywhere?

**Comment by lukasmwerner** [comment]
Uhhhhh I don't think so but we have tape

**Comment by pdxaeroman** [comment]
We are NOT taping the entire backside of a banner

**Comment by lukasmwerner** [comment]
we could glue it to some canvas or fabric
